test/unit/vl-map-delete-action.test.html

```diff
@@ -112,6 +112,47 @@
         assert.isTrue(removeFeatureStub.calledOnce);
         assert.deepEqual(removeFeatureStub.args[0][0], feature);
       });
+
+      test('nadat de delete actie voltooid is, kan er met de onDelete callback voor gezorgd worden dat bepaalde features verdwijnen en andere niet', async () => {
+        const map = fixture('vl-map-delete-action-fixture');
+        await map.ready;
+        const deleteAction = map.querySelector('vl-map-delete-action');
+        await awaitUntil(() => deleteAction.action != null);
+        const removeFeatureStub = sinon.stub();
+        sinon.stub(deleteAction.layer, 'getSource').returns({
+          removeFeature: removeFeatureStub,
+        });
+
+        deleteAction.onDelete((features, resolve, reject) => {
+          features.forEach((feature) => {
+            if (feature.verwijderen === 'JA') {
+              resolve(feature);
+            } else {
+              reject(feature);
+            }
+          });
+        });
+        const teVerwijderenFeature = {
+          getStyle: () => {},
+          setStyle: () => {},
+          getId: () => null,
+          verwijderen: 'JA',
+        };
+        const featureDieNietVerwijderdMagWorden = {
+          getStyle: () => {},
+          setStyle: () => {},
+          getId: () => null,
+          verwijderen: 'NEE',
+        };
```

Comment:
Kunt ge niet gewoon ne feature maken hier? Voelt voor mij hetzelfde aan als in Java domein objecten mocken ...

Reply:
Kan wel, maar dan zouden we die op een of andere manier moeten kunnen importen van OpenLayers. Zal eens zien of dat dat lukt.

Reply:
Ok ja, via vl-mapactions kan dit gewoon, zal dat dan voorzien